dxr_server/src/lib.rs

```diff
@@ -71,6 +71,7 @@ pub async fn server(handlers: HandlerMap, body: &str, headers: HeaderMap) -> (St
                     let handler = match handlers.get(name.as_str()) {
                         Some(handler) => handler,
                         None => {
+                            log::warn!("Missing rpc handler for {}", name);
```

Comment:
Two things here:

- By default I think lowering this to a `log::debug` would be better. Looking at invalid / unknown RPC calls would likely be something that you only do during development, and it could pollute logs during production if scrapers or bots find the server and start submitting bogus requests to it. 😬 
- Maybe the message would be better as `"No handler registered for method: {}", name` or something like that. Just because there is no handler registered for a method doesn't mean that it's *missing*, it might just be *wrong* (i.e. invalid).